thavalon-webapp/src/components/gameComponents/gameRoot.tsx

```diff
@@ -38,9 +39,13 @@ export function GameRoot(): JSX.Element {
                     <MissionResults />
                 </div>
             </div>
-            <div className="col-right">
+            <div className="col-center">
+                <GameHeader />
                 <PlayerBoard />
             </div>
+            <div className="col-right">
+                <h1>Right Col</h1>
+            </div>
```

Comment:
What's going to go here?

Reply:
Placeholder for the toast notifications. We needed a third column to keep the board centered and will use it for notifications.